<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Melter</class>
 <widget class="QWidget" name="Melter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>381</width>
     <height>116</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="rowValues">
      <property name="text">
       <string>Value Vars</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="columnValues">
      <property name="text">
       <string>Value Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pivotColumns">
      <property name="text">
       <string>ID Vars</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pivotWithColumns">
      <property name="text">
       <string>Value Vars</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Unpivot</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
